src/main/java/embl/ebi/variation/eva/pipeline/steps/tasklet/IndexesCreate.java

```diff
@@ -50,14 +49,9 @@
     public RepeatStatus execute(StepContribution contribution, ChunkContext chunkContext) throws Exception {
 
         ObjectMap pipelineOptions = variantJobsArgs.getPipelineOptions();
-        if (pipelineOptions.getBoolean(SKIP_INITIALIZE_INDICES)) {
-            logger.info("skipping indices creation step, initialize.indices.skip is set to {} ",
-                    pipelineOptions.getBoolean(SKIP_INITIALIZE_INDICES));
-        } else {
-            MongoOperations operations = MongoDBHelper.getMongoOperationsFromPipelineOptions(pipelineOptions);
-            operations.getCollection(pipelineOptions.getString("db.collections.features.name"))
-                    .createIndex(new BasicDBObject("name", 1), new BasicDBObject("sparse", true));
-        }
+        MongoOperations operations = MongoDBHelper.getMongoOperationsFromPipelineOptions(pipelineOptions);
+        operations.getCollection(pipelineOptions.getString("db.collections.features.name"))
+                .createIndex(new BasicDBObject("name", 1), new BasicDBObject("sparse", true).append("background", true));
```

Comment:
I think background must be set calling [createIndex(Bson, IndexOptions)](http://api.mongodb.com/java/3.0/com/mongodb/client/MongoCollection.html#createIndex-org.bson.conversions.Bson-com.mongodb.client.model.IndexOptions-). Not sure if that call to `append` will create an index on another field or what.